src/domain/Mypage/AccountSetting/AccountSetting.module.scss

```
@@ -15,12 +15,12 @@
 
   &__header {
     padding-bottom: 8px;
-    border-bottom: 1px solid f.color(orange-100);
+    border-bottom: 2px solid f.color(gray-300);
     margin-bottom: 16px;
     @include m.flex-box(space-between, center);
```

Stylelint (17.2.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)